README.md

```diff
@@ -860,6 +860,27 @@ of the SDK.
    npm start
    ```
 
+## Providing Custom Public Key
+
+By default, the SDK will download the public key from Descope's servers. You can also provide your own public key. This is useful when the server you are running the SDK on does not have access to the internet.
+
+You can find your public key in the `https://api.descope.com/v2/keys/<project-id>` endpoint.
```

Comment:
Maybe we can also add link to keys section in docs.

Reply:
good idea

added